src/vs/workbench/services/hover/browser/hoverService.ts

```diff
@@ -105,12 +105,16 @@ export class HoverService implements IHoverService {
 			hover.isLocked = true;
 			return;
 		}
+
 		const event = new StandardKeyboardEvent(e);
 		const keybinding = this._keybindingService.resolveKeyboardEvent(event);
 		if (keybinding.getSingleModifierDispatchParts().some(value => !!value) || this._keybindingService.softDispatch(event, event.target)) {
 			return;
 		}
-		this.hideHover();
+
+		if (e.key !== 'Tab') {
+			this.hideHover();
+		}
```

Comment:
Instead of this, I think we should still be hiding the hover but only when the `document.activeElement` lies outside the hover. Otherwise this happens which doesn't see right:

![Recording 2022-10-18 at 07 48 55](https://user-images.githubusercontent.com/2193314/196464689-17540247-2c69-4325-ae80-c7f09ad64e6b.gif)

Reply:
Actually, being able to tab into the hover seems pretty good, because I'm not sure how else a user would think to use the keyboard to navigate into the hover 
I also think that a tab loop might help, though that is a more complicated design. I can split the maxHeight changes into a separate PR.